pdb-alignment-api/src/main/java/org/cbioportal/pdb_annotation/web/controllers/AlignmentController.java

```diff
@@ -49,15 +46,11 @@ public AlignmentController(AlignmentRepository alignmentRepository, EnsemblRepos
     @RequestMapping(value = "/StructureMappingQuery", method = RequestMethod.GET, produces = "application/json")
     @ResponseBody
     @ResponseStatus(HttpStatus.OK)
-    public List<Alignment> getPdbAlignmentByEnsemblId(
+    public List<Alignment> getPdbAlignmentByEnsemblId (
             @RequestParam @ApiParam(value = "Input ensembl id. For example ENSP00000483207.2", required = true, allowMultiple = true) String ensemblId) {
-        List<Alignment> list = new ArrayList();
-        try {
-            List<Alignment> it = alignmentRepository.findByEnsemblId(ensemblId);
-            list.addAll(it);
-        } catch (Exception ex) {
-            ex.printStackTrace();
-        }
+        List<Alignment> it = alignmentRepository.findByEnsemblId(ensemblId);
```

Comment:
Ok, removed local list variables from controller.
Possibly, the intermediate result was to allow testing for a null value, but since that null value was not being handled / flagged / reported to user, I have simplified controllers.
Also, there were originally try/catch which suppressed Exceptions -- but I dropped that because we should not let an exception cause an inaccurate data return to the user.